Disable Kerberos by removing host keytab.

adelton opened the pull request:
#### Description:

- Disable Kerberos by removing host keytab.

#### Rationale:

- The key derivation function (KDF) in Kerberos is not FIPS compatible.

jan-cerny commented on linux_os/guide/services/kerberos/kerberos_disable_no_keytab/rule.yml:
```diff
@@ -0,0 +1,28 @@
+description: |-
+    Kerberos is not an approved key distribution method for
+    Common Criteria. To prevent using Kerberos by system daemons,
+    remove the host keytab <tt>/etc/krb5.keytab</tt>.
```
The description mentions specifically `/etc/krb5.keytab` but the OVAL uses a wildcard and checks any file in `/etc` which name ends with `.keytab`.

jan-cerny commented on linux_os/guide/services/kerberos/kerberos_disable_no_keytab/oval/shared.xml:
```diff
@@ -0,0 +1,25 @@
+  </definition>
+
+  <unix:file_object id="obj_kerberos_disable_no_keytab" version="1" comment="fapolicyd.mounts">
+    <unix:filepath operation="pattern match">/etc/*.keytab</unix:filepath>
```
The regular expression should be `^/etc/.*\.keytab$`.

jan-cerny commented on linux_os/guide/services/kerberos/kerberos_disable_no_keytab/oval/shared.xml:
```diff
@@ -0,0 +1,25 @@
+      <description>Check that there is no Kerberos keytab file present in /etc</description>
+    </metadata>
+    <criteria>
+      <criterion test_ref="test_kerberos_disable_no_keytab" negate="true"
```
I think instead of using negation, you can set `check="all" check_existence="none_exist"` in the `file_test` element and remove the `negate` attribute. However, your solution works for me as well.

jan-cerny commented:
@openscap-ci test this please

adelton commented:
Addressed the comments -> 7511e33a5c24f172edc409e20d7b7240d91541a2.

Sorry about that, I thought I had the patch updated and pushed before creating the PR but evidently I did not.

jan-cerny commented:
@openscap-ci test this please
